platform/accessibility/508-accessibility-best-practices.md

```diff
@@ -31,9 +34,14 @@
 1. Axe Scans
 
    1. Front-end engineering should install the [axe plugin for Chrome or Firefox](https://deque.com/axe) and run it periodically during their daily work.
```

Comment:
Is it Axe or aXe or axe?

Reply:
axe per the Deque site ( https://deque.com/axe ) so I'll go with that.